CSS- Networking updating NW Documentation (#126267)
Azure/azure-content: articles/network-watcher/packet-capture-overview.md

There's a limit of 10,000 parallel packet capture sessions per region per subscription. This limit applies only to the sessions and doesn't apply to the saved packet capture files either locally on the VM or in a storage account. See the [Network Watcher service limits page](../azure-resource-manager/management/azure-subscription-service-limits.md#azure-network-watcher-limits) for a full list of limits. 

> Packet capture currently supports SAS authentication when writing completed packet captures to storage accounts. SAS authentication is required if choosing to write to storage. If SAS authentication is not an option, then writing to the local file storage on the virtual machine where the packet capture is taken is the alternative.

Reviewer: ```suggestion
Packet capture currently supports SAS authentication when saving completed packet captures to a storage account. SAS authentication is required if choosing to save to a storage account. If SAS authentication isn't an option, then saving to the local disk on the virtual machine where the packet capture is taken is the alternative.
```

Author: Looks good
